orm/main.go

```diff
@@ -63,6 +63,40 @@ func createOrmServer() *martini.ClassicMartini {
 		writeChallenge(rs, challenge)
 	})
 
+	m.Put("/challenge/:id", binding.MultipartForm(ChallengeForm{}), func(rs http.ResponseWriter, rq *http.Request, f ChallengeForm, params martini.Params) {
+		id := params["id"]
+		challenge, err := orm.FindChallenge(id)
+
+		challenge.Title = f.Title
+		challenge.TimeLimit = f.TimeLimit
+		challenge.MemoryLimit = f.MemoryLimit
+
+		if err != nil {
+			utils.WriteResponse(rs, http.StatusInternalServerError, "Database error trying to find challenge with id "+string(id)+" to update", err)
+			return
+		}
+		inputsArray, err := parseRequestFiles(f.Inputs)
+		if err != nil {
+			utils.WriteResponse(rs, http.StatusInternalServerError, "Error trying to access input files", err)
```

Comment:
I really dislike bodies that only have a string as a response.
The response should be structured and parseable, such as a JSON. Doesn't matter if it's an error or a response.

What about creating a `utils.WriteJSONError()`?